Marsiple/App/Albums/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  PhotoCollectionViewCell.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class PhotoCollectionViewCell: UICollectionViewCell {
+    private let imageView = UIImageView.autolayoutView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupImageView()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension PhotoCollectionViewCell {
+    func updateCell(withImage image: UIImage) {
```

Comment:
The expected data type for the photo cells is a `URL`, not a `UIImage`.

Reply:
Shouldn't 'ViewController' do the business logic (fetching image), and cells are just presenting received data?